bubblewrap.c

```diff
@@ -32,9 +32,11 @@
 #include <sys/signalfd.h>
 #include <sys/capability.h>
 #include <sys/prctl.h>
+#include <sys/syscall.h>
 #include <linux/sched.h>
 #include <linux/seccomp.h>
 #include <linux/filter.h>
+#include <linux/landlock.h>
```

Comment:
What kernel/glibc versions, if any, guarantee to have this header?

The build system should probably check for this header with `cc.check_header('linux/landlock.h')` and define `HAVE_LANDLOCK_H`, so that this can become


```suggestion
#ifdef HAVE_LANDLOCK_H
#include <linux/landlock.h>
#endif
```

and then the use of Landlock features in the code can be something like (pseudocode)

```
if (opt_scope_abs_unix_sockets)
  {
#ifdef HAVE_LANDLOCK_H
    do Landlock things;
#else
    die ("Landlock not available at compile time, cannot implement --scope-abstract-unix-sockets");
#endif
  }
```